### ISSUE TITLE
[RFC] Cron Tasks

### ISSUE THREAD

**Issue by gildonei** [phpMv/ubiquity#55]
## Summary

Allow developers to execute Controller actions on shell, without renders

## Motivation

Shell tasks are a very commom need on projects that need to run routines to update status, reports, consolidate datas, and other stuff.

## Compatibility with Ubiquity's philosophy 

Indicate compatibility or improvements in:
- ease of use
- framework resources

## Expected results

Be possible to run controller actions through shell or command line without renders and keeping framework funcionalities like ACL and others.

## Additional context

Framework should be able to re-use models actions and framework actions almost like it was being executed on a web broser.

A sample case could be send a Birthday mail for Customers from an e-commerce

**Comment by UlasSAYGINIM** [comment]
is this only crontab issue? just controller can be called by crontab or other things?
is there any another examples from other projects?

**Comment by gildonei** [comment]
@UlasSAYGINIM yes, it is about using Ubiquity with cron tasks. I use to develop with Cake you can see about it in https://book.cakephp.org/4/en/console-commands.html

A sample of use could be run daily backups, scheduled jobs like financial data that have to run out of work time, send email notifications according specific situations like birthday, billing. etc.
